client/components/events/EventViewToggle/EventViewToggle.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { EventCalendarView } from "../EventCalendarView"
+import { EventListView } from "../EventListView/EventListView"
+import { Event } from '../../../context'
+import { colours, device, fontWeight, desktopFontSize, mobileFontSize } from '../../../styles'
+import styled from 'styled-components'
+
+enum View { List, Calendar }
```

Comment:
```suggestion
const enum View {
  List = 'list',
  Calendar = 'calendar',
}
```

slightly better for performance

https://www.typescriptlang.org/docs/handbook/enums.html#const-enums

I added in the string literals because I think they might be useful to stick into the URL query params (see comment below)